src/main/java/dev/aura/bungeechat/command/LocalToCommand.java

```diff
@@ -0,0 +1,57 @@
+package dev.aura.bungeechat.command;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import dev.aura.bungeechat.account.BungeecordAccountManager;
+import dev.aura.bungeechat.api.account.BungeeChatAccount;
+import dev.aura.bungeechat.command.BaseCommand;
+import dev.aura.bungeechat.message.Context;
+import dev.aura.bungeechat.message.Message;
+import dev.aura.bungeechat.message.MessagesService;
+import dev.aura.bungeechat.permission.Permission;
+import dev.aura.bungeechat.permission.PermissionManager;
+import net.md_5.bungee.api.CommandSender;
+import net.md_5.bungee.api.ProxyServer;
+import net.md_5.bungee.api.config.ServerInfo;
+import dev.aura.bungeechat.module.LocalToModule;
+
+public class LocalToCommand extends BaseCommand {
+
+	public LocalToCommand(LocalToModule localToModule) {
+		super("localto", localToModule.getModuleSection().getStringList("aliases"));
+	}
+	
+	@Override
+	public void execute(CommandSender sender, String[] args) {
+		
+		if (PermissionManager.hasPermission(sender, Permission.COMMAND_LOCALTO)) {
+			if (args.length < 2) {
+				MessagesService.sendMessage(sender, Message.INCORRECT_USAGE.get(sender, "/localto <server> <message>"));
+			}
+			else
+			{
+				String server = args[0];
+				String finalMessage = Arrays.stream(args).skip(1).collect(Collectors.joining(" "));
```

Comment:
Calculate message after verifying the server exists. I recommend an `if` with a `return` when the server doesn't exist. Same with all other method ending conditions.